Bangazon/Managers/PaymentTypeManager.cs

```diff
@@ -6,14 +6,18 @@ namespace BangazonCLI
     public class PaymentTypeManager
     {
         private dbUtilities _db;
+        private CustomerManager _cm;
         // Constructor method to establish a connection with the database, database conenction is passed in as an argument..
         public PaymentTypeManager(dbUtilities db)
         {
+            _cm = new CustomerManager(db);
             _db = db;
         }
         // Adds a new PaymentType--passed in as an argument--to the database
-        public int AddNewPaymentType(PaymentType newPaymentType){
-            return 4;
+        public int AddNewPaymentType(int newAccountNumber, string newAccountName){
```

Comment:
Something for team discussion... do we pass the individual values, or do we pass an entire paymentType object? My preference is for the latter. In this case a payment type only has two properties so it's pretty concise. But for something like adding a new customer? That's 7 properties. Easier to create a new customer (or paymentType, or whatever), and pass the entire object to AddNewCustomer or AddNewPaymentType.
Not a big deal in this PR, just something that we  should standardize one way or another later.